invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -14,77 +14,81 @@
   <div class="ui container relaxed grid page-subheader mr-0-mobile ml-0-mobile">
     <div class="row pb-0">
       <div class="sixteen wide mobile sixteen wide tablet thirteen wide computer column">
+        <div class="ui items">
```

Comment:
sorry - really hard to understand what was the real change here due to reformatting (if any - could you point out to the file and line you've changed?)

Reply:
The change basically revolved around wrapping the community header in a item from semantic ui. The desired result had a layout pretty similar to the item from semantic ui so used to to avoid custom css.